modules/orionode/lib/args.js

```diff
@@ -45,7 +45,9 @@ exports.createDirs = function(dirs, callback) {
 exports.readPasswordFile = function(passwordFile) {
 	if (passwordFile) {
 		try {
-			return fs.readFileSync(passwordFile,'utf8');
+			var lines = fs.readFileSync(passwordFile,'utf8')
+				.split('\n').filter(Boolean);
+			return lines.length>0?lines[0]:null;
```

Comment:
1. Yes. There should only be one line. Problem is that there seems to be a bug in the readFileSync function that appends an carriage return. I think it's safe to ignore any garbage after a carriage return regardless
2. filter Boolean [takes garbage](http://www.devign.me/javascript-tip-remove-falsy-items-out-of-an-array) out of the array. 
3. I am using it in the orion build for kubernetes